### DISCUSSION TITLE
Seq.App.Email app send an email with wrong link

### DISCUSSION THREAD

**Discussion by doggy8088** [datalust/seq-tickets#1919] [Help] [answered]
I installed `Seq.App.Email` app into my Seq. When I configured an Alert, the Email sent to me contains a wrong link which contains a `localhost:80` domain because I run Seq inside a Docker container.

![image](https://github.com/datalust/seq-tickets/assets/88981/c96a9036-9284-4cce-a8a8-878926b340d4)

How can I configure the domain name to my Seq installation? I don't want to use `localhost:80` in the email.

**Comment by KodrAus** [comment, marked as answer]
Hi @doggy8088 :wave:

This is something you can use the [canonical URI configuration setting](https://docs.datalust.co/docs/json-config#apicanonicaluri) for. It'll make all inbound links generated by Seq use that URI instead of whatever it happens to be listening on locally.
